measure_iops/src/main.rs

```diff
@@ -91,15 +91,15 @@ async fn main() -> Result<()> {
         read_only: false,
     };
 
-    let (guest, mut io) = Guest::new(None);
+    let (guest, io) = Guest::new(None);
     let guest = Arc::new(guest);
 
-    if let Some(iop_limit) = opt.iop_limit {
-        io.set_iop_limit(16 * 1024 * 1024, iop_limit);
+    if let Some(_iop_limit) = opt.iop_limit {
+        bail!("IOP limit is not implemented"); // XXX fix this once implemented
```

Comment:
should be `CrucibleError::Unsupported` instead

Reply:
Done in 25ec12c50f5d, although I'm ambivalent (the function is returning `anyhow::Result` already, and it's a CLI tool rather than an internal library)